frontend/src/Components/UserProfile.js

```diff
@@ -67,15 +68,22 @@ export default function UserProfile({ id }) {
+              {userProfile.user_id === user_id ? (
+                <h1 className="text-2xl">No posts to show.<A href={`/home`} className="hover:text-blue-500">Start creating your posts here.</A></h1>
+              ):<h1 className="text-2xl">No posts to show.</h1>}     
```

Comment:
1. Move the second sentence to a new line?
2. It's not obvious that's it's a link (so just removed the "hover:" part, don't know if that will work)
![image](https://user-images.githubusercontent.com/43996118/114903699-af366e00-9e34-11eb-92a0-74d9a2c939b7.png)

```suggestion
                <h1 className="text-2xl">No posts to show.
                <A href={`/home`} className="text-blue-500">Start creating your posts here.</A></h1>
              ):<h1 className="text-2xl">No posts to show.</h1>}     
```

Reply:
A rebase is required after this @amalaabraham

Reply:
@aitikgupta Updated the style! :)

![Screenshot (91)](https://user-images.githubusercontent.com/36256165/115027809-c7b09200-9ee1-11eb-8981-4ab80e665d80.png)